src/main/java/org/infernalstudios/shieldexp/init/ShieldDataLoader.java

```diff
@@ -83,12 +85,47 @@ public static void parse(ResourceLocation name, JsonObject data) {
             SHIELD_STATS.remove(key);
             SHIELD_STATS.put(key, stats);
 
-            if (!key.equals(ShieldExpansion.MOD_ID + ":default"))
+            if (!key.equals(DEFAULT_SHIELD_NAME))
                 Config.extendList(key);
```

Comment:
See suggestion above (lines 32-35)

For lines 75-89:
```diff
-       String key = name.toString();
-       if (ForgeRegistries.ITEMS.containsKey(name) || key.equals(DEFAULT_SHIELD_NAME)) {
+       if (ForgeRegistries.ITEMS.containsKey(name) || name.equals(DEFAULT_SHIELD_NAME)) {
            Map<String, Double> stats = new HashMap<>();
            stats.put("cooldownTicks", data.getAsJsonObject().get("cooldownTicks").getAsDouble());
            stats.put("speedFactor", data.getAsJsonObject().get("speedFactor").getAsDouble());
            stats.put("parryDamage", data.getAsJsonObject().get("parryDamage").getAsDouble());
            stats.put("parryTicks", data.getAsJsonObject().get("parryTicks").getAsDouble());
            stats.put("stamina", data.getAsJsonObject().get("stamina").getAsDouble());
            stats.put("blastResistance", data.getAsJsonObject().get("blastResistance").getAsDouble());
            stats.put("flatDamage", data.getAsJsonObject().get("flatDamage").getAsDouble());
-           SHIELD_STATS.remove(key);
-           SHIELD_STATS.put(key, stats);
+           SHIELD_STATS.remove(name);
+           SHIELD_STATS.put(name, stats);

-           if (!key.equals(DEFAULT_SHIELD_NAME))
-               Config.extendList(key);
+           if (!name.equals(DEFAULT_SHIELD_NAME))
+               Config.extendList(name.toString());
```

It's possible to maybe convert `Config.extendList` to take in a RL instead of a String? Not sure.

Reply:
If your config is easy to work with, I can make it happen. I am trying not to change too many semantics.